src/librustc_typeck/diagnostics.rs

```diff
@@ -139,19 +139,36 @@ and [RFC 809] for more details.
 "##,
 
 E0067: r##"
-The left-hand side of an assignment operator must be an lvalue expression. An
-lvalue expression represents a memory location and includes item paths (ie,
-namespaced variables), dereferences, indexing expressions, and field references.
+The left-hand side of a compound assignment expression (or compound assignment
```

Comment:
i didn't mean for you to write "expression (or operator)".  Just pick one, please.